rules_go_gapic/go_gapic_repositories.bzl

```diff
@@ -0,0 +1,98 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@bazel_gazelle//:deps.bzl", "go_repository")
+
+# List of Go dependencies for the GAPIC necessary for compiling the go_library
+# go_gapic_library. Colocated with go_gapic_repositories to keep the GAPIC deps
+# together.
+go_gapic_deps_list = [
```

Comment:
Please clarify why it is here, instead of the build bazel file? 

It is an interesting idea to put it here, and looks logical (like dependencies go together), but repositories.bzl and the actual list of default dependencies (i.e. this constant) for some particular rule have different scope. 

Stuff in `repositories.bzl` file is to be used globally for the whole WORKSPACE, while this `go_gapic_deps_list` defines the list of default dependencies for one particular rule. 

Notice that this constant is used only in one place (i..e one particular marco), and there is no potential to use it anywhere else.

Also, for public constants please use UPPER_CASE. 

Unless there is a strong reason to keep it here, please move this back to `_go_gapic_postprocessed_srcjar`.

Reply:
I moved it back into the go_gapic_library definition, where it was before. I thought putting dependency info together was a decent organization, but you are right! Thanks for the explanation.